src/service/url-expander/expander.js

```diff
@@ -236,7 +237,7 @@ export class Expander {
           });
     };
 
-    return evaluateNextLevel();
+    return evaluateNextLevel(/* encode */ !this.noEncode_);
```

Comment:
also change this.noEncode_  to this.encode_

Reply:
Done.